### PR TITLE
[QoL]: Adding a prettier config and run prettier on all files for consistency

### PR DESCRIPTION
**Pull request by benaduggan**
- Add prettier configuration and run on all files
- Updated Readme to use `npm start` as the command was renamed

I excluded the templates folder because it seems like that might be generated content? But I'm not sure if that's the case or not, let me know!

I just used a configuration that I've become used to, but definitely can change the rules if you think there would be options this community would prefer!

### CODE REVIEW

File: prettier.config.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const config = {
+  printWidth: 100,
+  arrowParens: 'always',
+  singleQuote: true,
+  tabWidth: 2,
```
Comment (osmarpetry) [inline prettier.config.js]: I believe they want 4 spaces, not 2 spaces on the files. Can you check if it is the pattern of almost the project?

File: package.json [hunk 2]
```diff
@@ -1,52 +1,54 @@
...
+    "server-reinstall": "rm -rf node_modules && rm -rf package-lock.json && npm install",
+    "server-logs": "sudo pm2 logs BikeTag",
+    "server-dev": "sudo pm2 start ecosystem.config.js --env dev",
+    "server-start": "sudo pm2 start ecosystem.config.js --env production",
+    "server-restart": "sudo pm2 restart ecosystem.config.js --env production",
+    "server-run": "sudo pm2 start ecosystem.config.js --env production",
+    "server-stop": "sudo pm2 stop BikeTag",
+    "server-rerun": "sudo pm2 delete BikeTag && sudo pm2 start ecosystem.config.js --env production",
+    "server-deploy": "npm run server-update && npm run server-restart && npm run server-logs",
+    "prettier-all": "git ls-files -X .prettierignore | grep -E '\\.(js|json|yml|md|html)$' | xargs yarn prettier --write"
```
Comment (osmarpetry) [inline package.json]: Why this big command to use prettier? Why not just call prettier alone? I got this error when I tried to run the command:

**Review by osmarpetry — CHANGES_REQUESTED** [review]
I believe they want 4 spaces, not 2 spaces on the files. Can you check if it is the pattern of almost the project?

File: package.json (hunk 2 above)
Comment (benaduggan) [reply package.json]: I just pulled it from another repo I had it set up in but that might have been a more specific use case! I changed it to be more simplified and recommitted. Can you check if you can run it now?

File: prettier.config.js (hunk 1 above)
Comment (benaduggan) [reply prettier.config.js]: I think it was mostly just using tab characters? so I'd imagine that might be set to 4 space tabs by default in other's editors. I'm cool with 4 space tabs, so I changed that and recommitted!

**Comment by KenEucker** [comment]
@benaduggan 

I like this, thank you. Consistency is important and I've never set up a prettier_config before so this is helpful.

**Pull request merged**